### DISCUSSION TITLE
Object Tracking Working Group

### DISCUSSION THREAD

**Discussion by Jegelewicz** [ArctosDB/arctos#7366] [Announcements]
From the AWG Issues Meeting yesterday, we need to complete [requirements for permissions](https://docs.google.com/spreadsheets/d/1Dha9tra2W_h8Rn9tJN6dSMXvy0Nl0Lc8dCLDt2FEN10/edit#gid=0) for Dusty, but we also have quite a few unresolved issues related to object tracking that need some focused attention, most notably the uniqueness of barcodes and how that should work across the database.

I have created a [When2Meet](https://www.when2meet.com/?23482543-TsU2Q) so that we can find a time to work through some of this. I'd like to plan for at least a two hour time slot (three would be even better) so that we can remove distractions and get some things done. Please add your availability by the end of the day on Monday, February 5, so that I can select a time to meet. I have started the [agenda](https://docs.google.com/document/d/1on0bbE7Z21YycA0CDeuyDmky96GW72ZcxpeMga29h2Y/edit#heading=h.pgimbtbq57uw) so you can see what I hope we can tackle.

@campmlc @falco-rk @megulbranson @Nicole-Ridgwell-NMMNHS @atrox10 @mkoo

**Comment by Jegelewicz** [comment]
Email sent to AWG Google Group

> AWG Member,
> 
> From the AWG Issues Meeting yesterday, we need to complete [requirements for permissions](https://docs.google.com/spreadsheets/d/1Dha9tra2W_h8Rn9tJN6dSMXvy0Nl0Lc8dCLDt2FEN10/edit#gid=0) for managing containers in object tracking, but we also have quite a few unresolved issues related to object tracking that need some focused attention, most notably the uniqueness of barcodes and how that should work across the database.
> 
> I have created a [When2Meet](https://www.when2meet.com/?23482543-TsU2Q) so that we can find a time to work through some of this. I'd like to plan for at least a two hour time slot (three would be even better) so that we can remove distractions and get some things done. If you are interested in object tracking and would like to participate, please add your availability by the end of the day on Monday, February 5, so that I can select a time to meet. I have started the [agenda](https://docs.google.com/document/d/1on0bbE7Z21YycA0CDeuyDmky96GW72ZcxpeMga29h2Y/edit#heading=h.pgimbtbq57uw) so you can see what I hope we can tackle.
> 
> Adios,
> 

Teresa J. Mayfield-Meyer

**Reply by dustymc** [reply]
https://github.com/ArctosDB/arctos/issues/6789 is resolved and done and shouldn't be open, I'll try to close it again....

**Reply by Jegelewicz** [reply]
So  does this mean that containers are now owned by an institution and we could have two barcodes = "1"?

**Comment by Jegelewicz** [comment]
> AWG Member,
> 
> The Object Tracking Working Group Met yesterday, and we made some progress related to permission roles for managing containers.
> I have created a [When2Meet](https://www.when2meet.com/?23905714-RiWuh) so that we can find a time to continue our conversation and see some demos of current users' workflows. If you are interested in object tracking and would like to participate, please add your availability by the end of the day on Friday, March 8, so that I can select a time to meet. You can review the rolling [agenda](https://docs.google.com/document/d/1on0bbE7Z21YycA0CDeuyDmky96GW72ZcxpeMga29h2Y/edit#heading=h.9o33lolz04r3) to see what we plan to discuss.
> 
> Adios,

Teresa J. Mayfield-Meyer

**Comment by happiah-madson** [comment]
Reminder that there is an Object Tracking Working Group meeting today at 11 am Eastern, 9 am Mountain.

https://northeastern.zoom.us/j/98285247981?pwd=sEsmsmVsHiMYRgba7kEdTEUx9kSYKM.1
